docs/platform/howto/byoc/create-custom-cloud.rst

```diff
@@ -0,0 +1,138 @@
+Create a custom cloud in Aiven |beta|
+=====================================
+
+Learn how to add a :doc:`custom cloud </docs/platform/concepts/byoc>` to your Aiven organization, and check out how it works.
+
+.. important::
+
+   * To create a custom cloud in your Aiven organization, you need the bring your own cloud (BYOC) feature enabled for this organization.
+   * BYOC is a beta feature, and its availability is limited. To enable BYOC for your Aiven organization, contact `sales@Aiven.io <mailto:sales@Aiven.io>`_.
+   * For more information on BYOC, see :doc:`Bring your own cloud </docs/platform/concepts/byoc>`. 
+
+About creating a custom cloud
+-----------------------------
+
+If you enabled BYOC and you have the administrator's role in your organization, you can create a custom cloud using `Aiven Console <https://console.aiven.io/>`_.
+
+Limitations
+'''''''''''
+
+* Administrator's role is required for creating custom clouds.
+* You need to use AWS as a cloud provider.
+* You need to use the :ref:`BYOC standard deployment <byoc-standard>` as a deployment model for your custom cloud.
+
+Prerequisites
+-------------
+
+* Administrator's role for your Aiven organization
+* BYOC feature enabled from `sales@Aiven.io <mailto:sales@Aiven.io>`_
+* Access to `Aiven Console <https://console.aiven.io/>`_
+
+Create a custom cloud
+---------------------
+
+Navigate to BYOC in Aiven Console
+'''''''''''''''''''''''''''''''''
+
+1. Log in to `Aiven Console <https://console.aiven.io/>`_ as an administrator.
+2. From the left sidebar, select **Bring you own cloud**.
+3. In the **Bring you own cloud** view, select **Create custom cloud**.
+
+Generate an infrastructure template
+'''''''''''''''''''''''''''''''''''
+
+In this step, an infrastructure-as-code (IaC) template is generated. For that purpose, AWS CloudFormation or Terraform can be used.
```

Comment:
Perhaps more accurate: "AWS CloudFormation and Terraform formats are supported for the tempalte"